### PR TITLE
Include a security policy for PDF.js

### PR DESCRIPTION
**Pull request by timvandermeij**
This makes sure that security researchers can find the required information for reporting security vulnerabilities in a standardized manner across GitHub repositories. Please refer to https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository for more information.

/cc @dveditz

### CODE REVIEW

**Review by calixteman — APPROVED** [review]
LGTM. Thank you.

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Security policy
+
+Mozilla takes the security of our software seriously. If you believe you have found a security vulnerability in PDF.js, please report it to us as described below.
+
+## Reporting security vulnerabilities
+
+**Please don't report security vulnerabilities through public GitHub issues.**
+
+Instead, please report security vulnerabilities in [Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=PDF%20Viewer&groups=firefox-core-security), making sure that the checkbox in the "Security" section is checked so the required access controls are automatically configured. The Mozilla security team will process the bug as described in [Mozilla's security bugs policy](https://www.mozilla.org/en-US/about/governance/policies/security-group/bugs).
```
Comment (calixteman) [inline .github/SECURITY.md:9]: Maybe we could add a screenshot
image
to help the user to see what we're talking about here. I know some people are sometimes a bit scared by the Bugzilla ui.

Comment (timvandermeij) [reply .github/SECURITY.md]: Good idea; I have included this in the new commit.

**Comment by timvandermeij** [comment]
You can find the rendered version here: https://github.com/mozilla/pdf.js/blob/2beae7aad6e9c89a794d3c87be21fcd5fe64e842/.github/SECURITY.md

**Pull request merged**